proposals/0228-fix-expressiblebystringinterpolation.md

```diff
@@ -88,6 +88,55 @@ String interpolation is the only remaining client of this type-checker entry poi
 
 </details>
 
+### Potential uses
+
+An improved string interpolation design could enable future standard library and Apple framework features; here are a few ideas. Note that we are not proposing these now and might not ever propose them in the future—they're just examples.
```

Comment:
This isn't restricted to Apple frameworks nor are Apple frameworks specially blessed, necessarily. How about this phrasing:

"An improved string interpolation design enables future functionally in the standard library as well as other frameworks and packages. Here are a few examples demonstrating the usefulness of our approach; note that these are for illustrative purposes only and we are not proposing any specific solution or syntax."